### PR TITLE
docs: add zh-CN translation

### PR DESCRIPTION
**Pull request by renbaoshuo**

### CODE REVIEW

File: docs/zh-cn/update.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+Ben 简单地说：“资金是与项目相关的，而不是当前的维护者。”
+
+我们一致同意，我们**绝对不希望**有能力触及现有资金，并在 Open Collective 的支持下制定了一个过渡计划。
+
+:::tip 旧集体
+现在有一个 [fakerjs-legacy](https://opencollective.com/fakerjs-legacy) 集体。Open Collective 将所有资金转入遗留账户，并邀请 Marak 和另一位维护者 Brian 加入遗留集体，以便他们可以用这些资金做任何他们想做的事情。
+:::
+
+资金转移后，我们被邀请成为 Faker 集体的管理员。 这意味着我们保留了 Faker 集体的现有赞助商，他们为项目的持续维护付费。
```
Comment (poyea) [inline docs/zh-cn/update.md:131]:
```suggestion
资金转移后，我们被邀请成为 Faker 集体的管理员。 这意味着我们保留了 Faker 集体的现有赞助者，他们曾为项目的持续维护付费。
```

File: docs/zh-cn/update.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+我们认为这是最公平、最透明的行动方式。 该项目的发起人能够继续支持该项目及其持续发展。 Marak 和 Brian 能够保留之前捐赠给该项目的 11652.69 美元。
+
+由于 Open Collective 支持资金透明度，因此您可以访问 fakerjs-legacy 集体的交易以查看存款和任何其他已进行的交易。
+
+::: tip 赞助 Marak
+对于那些通过赞助 Faker 集体来赞助 Marak 的个人或组织，请捐赠到 [fakerjs-legacy 集体](https://opencollective.com/fakerjs-legacy) 或 Marak 的其他公开集体，我们不知道哪个是他的首选。如果我们收到 Marak 的其他消息，我们将更新此通知并发布新链接。
+
+我们团队也将在接下来的几天内发送电子邮件，通知 Open Collective 赞助商维护团队发生了变化。
```
Comment (poyea) [inline docs/zh-cn/update.md:140]:
```suggestion
我们团队也将在接下来的几天内发送电子邮件，通知 Open Collective 赞助者有关于维护团队发生的变化。
```

File: docs/zh-cn/update.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+
+Faker 的资金支持由 [oscollective.org](https://www.oscollective.org) 代表项目持有，并在 [opencollective.com/fakerjs](https://opencollective.com/fakerjs) 上管理，因此在 2022 年 1 月 12 日，我们联系了 [Open Source Collective 的执行董事](https://twitter.com/BenJam)……
+
+在与 Ben 的谈话中，他和我一起讨论了 Open Collective 的条款和条件。
+
+Ben 简单地说：“资金是与项目相关的，而不是当前的维护者。”
+
+我们一致同意，我们**绝对不希望**有能力触及现有资金，并在 Open Collective 的支持下制定了一个过渡计划。
+
+:::tip 旧集体
```
Comment (poyea) [inline docs/zh-cn/update.md:127]: 集体 → collective?

File: docs/zh-cn/update.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+我们没有关于他的 GitHub 赞助商的信息。
+:::
+
+## 团队将如何在财务上运作？
+
+我们联系了 Open Collective 的执行董事 Ben Nickolls 寻求建议。
+
+他有两个建议：
+
+1. 做 ESLint 正在做的事情。《[A year of paying contributors: Review](https://eslint.org/blog/2020/10/year-paying-contributors-review)？
```
Comment (poyea) [inline docs/zh-cn/update.md:151]:
```suggestion
1. 做 ESLint 正在做的事情。《[A year of paying contributors: Review](https://eslint.org/blog/2020/10/year-paying-contributors-review)》
```

**Comment by renbaoshuo** [comment]
感谢指正。

因为本人英语水平并不是很好，所以部分内容参考了第三方翻译工具所给出的结果，如有错误请见谅。

**Comment by poyea** [comment]
> 感谢指正。
> 
> 因为本人英语水平并不是很好，所以部分内容参考了第三方翻译工具所给出的结果，如有错误请见谅。

It's alright. I'll take a look again soon

File: docs/zh-cn/api/address.md [hunk 5]
```diff
@@ -0,0 +1,187 @@
+# 地址
+
+[[toc]]
+
+## 城市
+
+::: v-pre
+
+生成随机的本地化城市名称。格式字符串可以包含使用用 `{{}}` 包装的 faker 提供的任何方法，例如 `{{name.firstName}}` 以构建城市名称。
```
Comment (poyea) [inline docs/zh-cn/api/address.md:9]:
```suggestion
生成随机的本地化城市名称。格式字符串可以使用 `{{}}` 包含 Faker 提供的任何函数，例如 `{{name.firstName}}` 以构建城市名称。
```

File: docs/zh-cn/api/address.md [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+:::
+
+```js
+faker.address.city(); // Lake Raoulfort
+faker.address.city('{{name.lastName}}{{address.citySuffix}}'); // Powlowski port
+```
+
+## 城市前缀
+
+返回一个随机的本地化城市前缀
```
Comment (poyea) [inline docs/zh-cn/api/address.md:28]: Consistency
```suggestion
返回一个随机的本地化城市前缀。
```

File: docs/zh-cn/api/company.md [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+
+返回一个随机的标语形容词。
+
+```js
+faker.company.catchPhraseAdjective(); // Down-sized
+```
+
+## 标语描述符
+
+返回一个随机的标语描述符
```
Comment (poyea) [inline docs/zh-cn/api/company.md:66]:
```suggestion
返回一个随机的标语描述符。
```

File: docs/zh-cn/api/company.md [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+
+返回一个随机的标语名词。
+
+```js
+faker.company.catchPhraseNoun(); // complexity
+```
+
+## 商业研究
+
+返回一个随机的商业研究
```
Comment (poyea) [inline docs/zh-cn/api/company.md:82]:
```suggestion
返回一个随机的商业研究。
```

File: docs/zh-cn/api/company.md [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+
+```js
+faker.company.bs(); // ubiquitous empower e-business
+```
+
+## 商业研究形容词
+
+返回一个随机的商业研究形容词
```
Comment (poyea) [inline docs/zh-cn/api/company.md:90]:
```suggestion
返回一个随机的商业研究形容词。
```

File: docs/zh-cn/api/company.md [hunk 10]
```diff
@@ -0,0 +1,110 @@
...
+
+```js
+faker.company.bsAdjective(); // granular
+```
+
+## 商业研究流行语
+
+返回一个随机的商业研究流行语
```
Comment (poyea) [inline docs/zh-cn/api/company.md:98]:
```suggestion
返回一个随机的商业研究流行语。
```

File: docs/zh-cn/api/company.md [hunk 11]
```diff
@@ -0,0 +1,110 @@
...
+
+```js
+faker.company.bsBuzz(); // facilitate
+```
+
+## 商业研究名词
+
+返回一个随机的商业研究名词
```
Comment (poyea) [inline docs/zh-cn/api/company.md:106]:
```suggestion
返回一个随机的商业研究名词。
```

File: docs/zh-cn/api/database.md [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+
+返回一个随机的数据库编码。
+
+```js
+faker.database.collation(); // cp1250_general_ci
+```
+
+## 引擎
+
+返回一个随机的数据库引擎。
```
Comment (poyea) [inline docs/zh-cn/api/database.md:31]:
```suggestion
返回一个随机的数据库引擎名称。
```

File: docs/zh-cn/api/finance.md [hunk 13]
```diff
@@ -0,0 +1,167 @@
...
+
+## 以太坊地址 <Badge text="5.0.0+" type="tip" vertical="middle"/>
+
+返回一个随机的以太坊地址。
+
+```js
+faker.finance.ethereumAddress();
+```
+
+## IBAN（国际银行帐号）
```
Comment (poyea) [inline docs/zh-cn/api/finance.md:146]:
```suggestion
## 国际银行帐号（IBAN）
```

File: docs/zh-cn/api/finance.md [hunk 14]
```diff
@@ -0,0 +1,167 @@
...
+
+返回一个随机的 IBAN（国际银行帐号）。
```
Comment (poyea) [inline docs/zh-cn/api/finance.md:148]:
```suggestion
返回一个随机的 国际银行帐号（IBAN）。
```

File: docs/zh-cn/api/finance.md [hunk 15]
```diff
@@ -0,0 +1,167 @@
...
+| -------- | ------- | :-----: |
+| formated | boolean | `false` |
+:::
+
+```js
+faker.finance.iban(); // BE56302102061244
+faker.finance.iban(true); // GI29 MRCH LR1V 0284 KE24 6K8
+```
+
+## BIC（银行识别码）
```
Comment (poyea) [inline docs/zh-cn/api/finance.md:161]:
```suggestion
## 银行识别码（BIC）
```

File: docs/zh-cn/api/finance.md [hunk 16]
```diff
@@ -0,0 +1,167 @@
...
+
+返回一个随机的 BIC（银行识别码）。
```
Comment (poyea) [inline docs/zh-cn/api/finance.md:163]:
```suggestion
返回一个随机的 银行识别码（BIC）。
```

File: docs/zh-cn/api/helpers.md [hunk 17]
```diff
@@ -0,0 +1,262 @@
+# 辅助函数
+
+[[toc]]
+
+## 数组中的随机元素
+
+从提供的数组中返回一个随机值
```
Comment (poyea) [inline docs/zh-cn/api/helpers.md:7]:
```suggestion
从提供的数组中返回一个随机值。
```

**Review by poyea — APPROVED** [review]
Just a few comments. Otherwise it looks good to me

**Comment by renbaoshuo** [comment]
Fixed.

**Review by Shinigami92 — DISMISSED** [review]
I'm not educated in chinese, but for a start I think this is better than nothing
So I'm really happy about that we have such and I trust the contributors

**Review by Shinigami92 — DISMISSED** [review]

**Comment by ST-DDT** [comment]
This needs to be rebased. There are both conflicts and the layout + content of the docs have changed drastically.

Any idea on how to ensure, that the translated api docs are always up to date (e.g. fill in missing/new methods)

**Comment by poyea** [comment]
@ST-DDT I can take up the role to maintain all these. Plus zh-TW and zh-HK. (or zh-Hans and zh-Hant). I'm proficient in all of them. @renbaoshuo please do a rebase as the project contains some changes.

**Comment by renbaoshuo** [comment]
Sorry for my late reply.

But there is a problem, I don't know how to add i18n support to TypeDoc.

**Comment by ST-DDT** [comment]
> Sorry for my late reply.
> 
> But there is a problem, I don't know how to add i18n support to TypeDoc.

No problem. I think we don't have to use typedocs to genereate the translated documentation.
We can just manually fill these with custom content.

I work on https://github.com/faker-js/faker/pull/446 on the side to make it easier to reuse some parts in translated api docs.

Instead of having to write the entire document from scratch you just have to insert the actual translatabled content in there.
In the current stage this has to be done roughly like this:

````ts
const apiDocsTranslation = clone(originalApiDocs);
apiDocs[0 /* random*/].description = 'foo';
apiDocs[0 /* random*/].parameters[0 /* paramA */ ].description = 'bar';

<ApiMethod v-for="method of apiDocsTranslation" :method="method" />
````

But I also considered separating the texts from the api structure, which would simplify the api translations.

````ts
apiDocs.random= 'foo';
apiDocs.random.parameters.paramA= 'bar';
````

(Currently I'm not sure which values actually needs to be translated)

The later has the added benefit of allowing actual translation tools to be used (e.g. Crowdin).

**Comment by ST-DDT** [comment]
**Maintainer decision:**
For now, we only consider supporting docs translations, that we can check and update ourselves.

Nevertheless, thank you for your contributions.

**Pull request closed without merging**